providers/gce/gce_loadbalancer_external.go

```diff
@@ -881,6 +903,13 @@ func (g *Cloud) firewallNeedsUpdate(name, serviceName, ipAddress string, ports [
 	if !sourceRanges.Equal(actualSourceRanges) {
 		return true, true, nil
 	}
+
+	destinationRanges := []string{ipAddress}
+
+	if !reflect.DeepEqual(destinationRanges, fw.DestinationRanges) {
```

Comment:
this comparison is a bit awkward since destination ranges only has an IP address, no?
```
if len(fw.DestinationRanges) == 1 && fw.DestinationRanges[0] == ipAddress
```

since this seems to be pulled from intree we can follow up later